src/masternode-sync.cpp

```diff
@@ -23,13 +23,13 @@ CMasternodeSync masternodeSync;
 void CMasternodeSync::Fail()
 {
     nTimeLastFailure = GetTime();
-    nRequestedMasternodeAssets = MASTERNODE_SYNC_FAILED;
+    nCurrentAsset = MASTERNODE_SYNC_FAILED;
 }
 
 void CMasternodeSync::Reset()
 {
-    nRequestedMasternodeAssets = MASTERNODE_SYNC_INITIAL;
-    nRequestedMasternodeAttempt = 0;
+    nCurrentAsset = MASTERNODE_SYNC_INITIAL;
+    nPeerCount = 0;
```

Comment:
On first sight, it sounds like "peers currently in progress" or something like that. Maybe `nTriedPeerCount` is better?

Reply:
Agree, `nPeerCount` is confusing imo. Smth like `nTriedPeerCount` or simply `nTriedPeers` would be better I think.

Reply:
I probably don't understand the code well enough then, but `nTriedPeers` would seem misleading, right? As I understand it, it's the count of peers with which we are trying to sync a particular asset at the given moment, right? So "nTryingPeers" would seem more appropriate, but I have a feeling my understanding might be off.

@UdjinM6 Can you explain to me (since I seem to be confused) what exactly this variable is keeping count of? Is my understanding in the previous paragraph not correct?